src/viewer.js

```diff
@@ -1680,6 +1699,18 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return this;
     },
 
+    /**
+     * Force the viewer to reset it's size to it's container.
+     * @param Boolean ignoreAutorResizeSetting (default: false) If true, forces the resize even if Viewer.autoResize==false
+     * @returns {OpenSeadragon.Viewer} Chainable.
+     */
+     forceResize: function(ignoreAutoResizeSetting) {
+        if(ignoreAutoResizeSetting || this.autoResize){
+            this._onViewerResize(this, _getSafeElemSize(this.container));
+        }
+        return this;
```

Comment:
I know we have a lot of chainable functions, but I'm wanting to move away from that style, since it doesn't seem terribly useful in the OSD context, and we already have such a hodgepodge of chainable versus not. So, for new APIs, I suggest we don't add it. What do you think?

Reply:
I have no strong feelings either way, so I've removed that.